tests/ttnn/unit_tests/operations/eltwise/backward/utility_funcs.py

```diff
@@ -180,6 +180,20 @@ def compare_pcc(tt_tensor, golden_tensor, pcc=0.99):
     return status
 
 
+def compare_equal_all_close(tt_tensor, golden_tensor):
```

Comment:
Why...? Is there ever a situation where `comp_equal` would return `True` while `comp_allclose` would not?